scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/web/forms/DatasetForm.java

```diff
@@ -0,0 +1,52 @@
+package edu.asu.diging.scriptoocloud.web.forms;
+
+import javax.validation.constraints.NotEmpty;
+
+public class DatasetForm {
```

Comment:
what's the different between this and `DatasetEditForm`? You can probably use the same class.